### DISCUSSION TITLE
two nodes HA

### DISCUSSION THREAD

**Discussion by sulanta** [k3s-io/k3s#3534] [Q&A] [answered]
I saw the release note of v1.20 mentioned as below. Does this meant we can acheive HA on two nodes? if it is could you point to some documentation?

v1.20 - December 2020
Support a 2-node hot/cold HA model by leveraging etcd's learner node feature

**Comment by ChristianCiach** [comment, marked as answer]
This has been clarified here: https://github.com/k3s-io/k3s/pull/3431#discussion_r657692400

> upstream etcd doesn't support this yet. At the moment you can still have only a single learner that is on a short path to full voting member. Multiple learner is documented as being a 3.5 feature, but I have not seen any code for it appear yet:
https://etcd.io/docs/v3.3/learning/learner/#features-in-v35
